### DISCUSSION TITLE
Wingsuit Tutorial

### DISCUSSION THREAD

**Discussion by hughjward** [peterdsharpe/AeroSandbox#124] [Q&A] [answered]
Hello!

I am not sure if i am going crazy but I thought I remember from quite a long time ago a tutorial where there was a wingsuiter flying over a mountain. I thought it was along side the quadcopter tutorial in:

AeroSandbox/tutorial/03 - Trajectory Optimization and Optimal Control/03 - The AeroSandbox Dynamics Stack

Has it been removed?

Cheers
Hugh

**Comment by peterdsharpe** [comment, marked as answer]
Hi Hugh,

It's still here!
https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/03%20-%20Trajectory%20Optimization%20and%20Optimal%20Control/03%20-%20The%20AeroSandbox%20Dynamics%20Stack/05%20-%202D%20Aircraft%20Dynamics%20for%20Maneuvers.ipynb

**Comment by hughjward** [comment]
TLDR; yes i went crazy, off to make more coffee... 

Thanks!
